PeerioTechnologies/peerio-desktop
# feat: closure notices

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/18485/desktop-app-load-notice-of-closure
https://app.clubhouse.io/peerio/story/18521/desktop-service-outage-forever-banner
https://app.clubhouse.io/peerio/story/18524/desktop-account-settings-upgrade-changes
https://app.clubhouse.io/peerio/story/18475/desktop-profile-settings-upgrade-changes
https://app.clubhouse.io/peerio/story/18519/desktop-top-drawer-notice-in-files-tab
https://app.clubhouse.io/peerio/story/18517/desktop-sign-up-notice-of-closure
https://github.com/PeerioTechnologies/peerio-icebear/pull/378

Figma https://www.figma.com/file/f8wf0Mm4m91o6hpN6Joaurjl/18363%3A-In-app-banner-announcing-closure-of-service?node-id=13%3A1

#### Testing instructions
Closure notices *everywhere*

* redesign of NewUser screen (open app with no saved user)
* banner on Login screen
* banner on first step of CreateAccount
* full-page overlay every time you log in (including account creation)
* banner on Chats view
* banner in Settings view, which cannot be dismissed
  * note: on Figma it still shows the "Dismiss" button but there's a conversation between me and Kristina where she said dismiss should not be allowed in the Settings banner 
* "Upgrade" in main menu changed to "Peerio Closure"

NOTE: since we made this so quickly there are some inconsistencies in the documentation/feature requests... I erred on the side of Clubhouse being correct, followed by Figma.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
